bin/user_config_converter.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+
+from enum import Enum, auto
+import importlib
+import json
+import os
+from pathlib import Path
+import sys
+from typing import Type
+
+from fusion_engine_client.messages.configuration import PlatformStorageDataMessage
+from fusion_engine_client.parsers.decoder import FusionEngineDecoder
+
+# Add the parent directory to the search path to enable p1_runner package imports when not installed in Python.
+repo_root = Path(os.path.dirname(__file__)).parent.absolute()
+sys.path.append(str(repo_root))
+
+# Example version imported just to help with type checking
+from user_config_loaders.platform_1.version_7_1.user_config_loader.user_config_loader import UserConfig as UserConfigTypingClass
```

Comment:
I'd suggest we name the dirs `platform_type_1` if we want it to be numeric. `platform_1` is a bit unclear.